### ISSUE TITLE
go these errors after running the script is it OK and shall i restart the system

### ISSUE THREAD

**Issue by emad2333** [clong/MakeWindows10GreatAgain#8]
Access is denied. (Exception from HRESULT: 0x80070005 (E_ACCESSDENIED))
At C:\WINDOWS\system32\MakeWindows10GreatAgain\MakeWindows10GreatAgain.ps1:33 char:208
+ ... , '') -match 'From "Start" UnPin|Unpin from Start'} | %{$_.DoIt()}  }
+                                                             ~~~~~~~~~
    + CategoryInfo          : OperationStopped: (:) [], UnauthorizedAccessException
    + FullyQualifiedErrorId : System.UnauthorizedAccessException

Access is denied. (Exception from HRESULT: 0x80070005 (E_ACCESSDENIED))
At C:\WINDOWS\system32\MakeWindows10GreatAgain\MakeWindows10GreatAgain.ps1:33 char:208
+ ... , '') -match 'From "Start" UnPin|Unpin from Start'} | %{$_.DoIt()}  }
+                                                             ~~~~~~~~~
    + CategoryInfo          : OperationStopped: (:) [], UnauthorizedAccessException
    + FullyQualifiedErrorId : System.UnauthorizedAccessException

Access is denied. (Exception from HRESULT: 0x80070005 (E_ACCESSDENIED))
At C:\WINDOWS\system32\MakeWindows10GreatAgain\MakeWindows10GreatAgain.ps1:33 char:208
+ ... , '') -match 'From "Start" UnPin|Unpin from Start'} | %{$_.DoIt()}  }
+                                                             ~~~~~~~~~
    + CategoryInfo          : OperationStopped: (:) [], UnauthorizedAccessException
    + FullyQualifiedErrorId : System.UnauthorizedAccessException

Access is denied. (Exception from HRESULT: 0x80070005 (E_ACCESSDENIED))
At C:\WINDOWS\system32\MakeWindows10GreatAgain\MakeWindows10GreatAgain.ps1:33 char:208
+ ... , '') -match 'From "Start" UnPin|Unpin from Start'} | %{$_.DoIt()}  }
+                                                             ~~~~~~~~~
    + CategoryInfo          : OperationStopped: (:) [], UnauthorizedAccessException
    + FullyQualifiedErrorId : System.UnauthorizedAccessException

Access is denied. (Exception from HRESULT: 0x80070005 (E_ACCESSDENIED))
At C:\WINDOWS\system32\MakeWindows10GreatAgain\MakeWindows10GreatAgain.ps1:33 char:208
+ ... , '') -match 'From "Start" UnPin|Unpin from Start'} | %{$_.DoIt()}  }
+                                                             ~~~~~~~~~
    + CategoryInfo          : OperationStopped: (:) [], UnauthorizedAccessException
    + FullyQualifiedErrorId : System.UnauthorizedAccessException

Access is denied. (Exception from HRESULT: 0x80070005 (E_ACCESSDENIED))
At C:\WINDOWS\system32\MakeWindows10GreatAgain\MakeWindows10GreatAgain.ps1:33 char:208
+ ... , '') -match 'From "Start" UnPin|Unpin from Start'} | %{$_.DoIt()}  }
+                                                             ~~~~~~~~~
    + CategoryInfo          : OperationStopped: (:) [], UnauthorizedAccessException                                         + FullyQualifiedErrorId : System.UnauthorizedAccessException                                                                                                                                                                                Access is denied. (Exception from HRESULT: 0x80070005 (E_ACCESSDENIED))                                                 At C:\WINDOWS\system32\MakeWindows10GreatAgain\MakeWindows10GreatAgain.ps1:33 char:208
+ ... , '') -match 'From "Start" UnPin|Unpin from Start'} | %{$_.DoIt()}  }
+                                                             ~~~~~~~~~
    + CategoryInfo          : OperationStopped: (:) [], UnauthorizedAccessException
    + FullyQualifiedErrorId : System.UnauthorizedAccessException

Access is denied. (Exception from HRESULT: 0x80070005 (E_ACCESSDENIED))
At C:\WINDOWS\system32\MakeWindows10GreatAgain\MakeWindows10GreatAgain.ps1:33 char:208
+ ... , '') -match 'From "Start" UnPin|Unpin from Start'} | %{$_.DoIt()}  }
+                                                             ~~~~~~~~~
    + CategoryInfo          : OperationStopped: (:) [], UnauthorizedAccessException
    + FullyQualifiedErrorId : System.UnauthorizedAccessException

Access is denied. (Exception from HRESULT: 0x80070005 (E_ACCESSDENIED))
At C:\WINDOWS\system32\MakeWindows10GreatAgain\MakeWindows10GreatAgain.ps1:33 char:208
+ ... , '') -match 'From "Start" UnPin|Unpin from Start'} | %{$_.DoIt()}  }
+                                                             ~~~~~~~~~
    + CategoryInfo          : OperationStopped: (:) [], UnauthorizedAccessException
    + FullyQualifiedErrorId : System.UnauthorizedAccessException

Access is denied. (Exception from HRESULT: 0x80070005 (E_ACCESSDENIED))
At C:\WINDOWS\system32\MakeWindows10GreatAgain\MakeWindows10GreatAgain.ps1:33 char:208
+ ... , '') -match 'From "Start" UnPin|Unpin from Start'} | %{$_.DoIt()}  }
+                                                             ~~~~~~~~~
    + CategoryInfo          : OperationStopped: (:) [], UnauthorizedAccessException
    + FullyQualifiedErrorId : System.UnauthorizedAccessException

Access is denied. (Exception from HRESULT: 0x80070005 (E_ACCESSDENIED))
At C:\WINDOWS\system32\MakeWindows10GreatAgain\MakeWindows10GreatAgain.ps1:33 char:208
+ ... , '') -match 'From "Start" UnPin|Unpin from Start'} | %{$_.DoIt()}  }
+                                                             ~~~~~~~~~
    + CategoryInfo          : OperationStopped: (:) [], UnauthorizedAccessException
    + FullyQualifiedErrorId : System.UnauthorizedAccessException

Access is denied. (Exception from HRESULT: 0x80070005 (E_ACCESSDENIED))
At C:\WINDOWS\system32\MakeWindows10GreatAgain\MakeWindows10GreatAgain.ps1:33 char:208
+ ... , '') -match 'From "Start" UnPin|Unpin from Start'} | %{$_.DoIt()}  }
+                                                             ~~~~~~~~~
    + CategoryInfo          : OperationStopped: (:) [], UnauthorizedAccessException
    + FullyQualifiedErrorId : System.UnauthorizedAccessException

**Comment by clong** [comment]
Did you run this as administrator?

**Comment by emad2333** [comment]
yes and after i put my pc to sleep  and entered my password it didnt recognise my password because my microsoft account was deleted due to the script

I had to recover my pc
